src/en/pin/pin-8.md

```diff
@@ -1,11 +1,13 @@
-This pin doubles up as the UART transmit pin, TX. It's also commonly known as "Serial" and, by default, will output a Console from your Pi that, with a suitable Serial cable, you can use to control your Pi via the command-line.
+This pin doubles up as the UART transmit pin, TX. It's also commonly known as "Serial" and, by default, will output a Console from your Pi that, with a [suitable Serial cable](https://elinux.org/RPi_Serial_Connection), you can use to control your Pi via the command line.
```

Comment:
I _think_ this might not actually be the default behaviour any more (because it requires setting the SoC to a fixed clockspeed, rather than letting it turbo on demand), and you need to explicitly request this functionality using `raspi-config`?

Reply:
Is that so? I don't mind lying temporarily to get this merged, and then figuring this out after the fact.

I uh... have been very, very deep in RP2040 lately so my Pi know-how is rusty.

Reply:
> Is that so? I don't mind lying temporarily to get this merged, and then figuring this out after the fact.

See https://www.raspberrypi.com/documentation/computers/configuration.html#interfacing-options and https://www.raspberrypi.com/documentation/computers/config_txt.html#enable_uart
(although this is often a moving target, and unfortunately the documentation isn't always kept up to date with the changes in Raspberry Pi OS)

> I uh... have been very, very deep in RP2040 lately so my Pi know-how is rusty.

Nice! :grinning: